module/command_tree_generator.py

```diff
@@ -1,29 +1,22 @@
 import json
 import inspect
+import re
 
 import common.frames
 
-from module.command_node import NodeType, Node
+from module.command_node import Node, Command
 from module.frame_functions import get_frames_with_description
 
+CAMEL_REGEX = re.compile("(?!^)([A-Z]+)")
```

Comment:
Not being an expert in regex i dont understand this line could you add documentation on this